handle/UnitGroup.lua

```diff
@@ -0,0 +1,21 @@
+require('CeresStdLib.base.Native')
+
+local __u
+
+local function tempEnumUnit()
+    return __u
+end
+
+replaceNative('GetEnumUnit', tempEnumUnit)
+replaceNative('ForGroup', function(g, func)
+    local i = 0
+    local j = BlzGroupGetSize(g)
+    local u = __u
+
+    while (i < j) do
+        __u = BlzGroupUnitAt(g, i)
+        pcall(func)

```

Comment:
pcall _returns_ error if it happens. Use `execute` from base.Execute to handle errors, so that catched errors are displayed.
But I don't recall `ForGroup` managing errors, so I suggest removing `pcall` from here.

Reply:
Will do. (Calling function directly instead of execute)